cmake/CeleritasAddTest.cmake

```diff
@@ -407,10 +407,16 @@ function(celeritas_add_test SOURCE_FILE)
       # Launch with MPI directly
       _celeritasaddtest_mpi_cmd(_test_cmd "${_np}" "${_EXE_NAME}")
 
-      set(_test_args "${_EXE_ARGS}")
+      set(_test_args)
```

Comment:
There is no `_EXE_ARGS` defined; that was just a leftover artifact from some previous iteration